opentelemetry-api/src/opentelemetry/context/__init__.py

```diff
@@ -68,6 +69,18 @@ def wrapper(  # type: ignore[misc]
     return typing.cast(_F, wrapper)  # type: ignore[misc]
 
 
+def create_key(keyname: str) -> str:
+    """To allow cross-cutting concern to control access to their local state,
+    the RuntimeContext API provides a function which takes a keyname as input,
+    and returns a unique key.
+    Args:
+        keyname: The key name is for debugging purposes and is not unique.
+    Returns:
+        A unique string representing the newly created key.
+    """
+    return str(uuid.uuid4())
```

Comment:
can this be a combination of the unique identifier and the provided key name to enable the debugging use cases mention in the doc string?

Reply:
Since the uuid returns a string like `'16fd2706-8baf-433b-82eb-8c7fada847da'`, would it be recommended to append the keyname and the uuid with another hyphen?

```
    return keyname + '-' + str(uuid.uuid4())
```
This would make the key
```
>>> create_key('examplename')
examplename-16fd2706-8baf-433b-82eb-8c7fada847da'
```

Reply:
good point! Updated to reflect this.